LL/dal_corpri_arad.trig

```diff
@@ -260,27 +206,27 @@
     a foaf:Person;
     irishRel:genName "Lugdach";
     irishRel:nomName "Lugdaid";
-    rel:childOf <#Labrada>.
+    rel:childOf <http://example.com/LL/de_genelach_dail_chormaic_&_h-úa_labrada.trig#Labrada>.
 
```

Comment:
We haven't had rel:childOf etc. with the object URL in a separate file before. Will this work? If so, no problem.

Reply:
There will still be a DCA (much reduced) and this was to maintain consistency between what will remain in DCA and the new file